### PR TITLE
support deployments to development

### PR DESCRIPTION
**Pull request by ayan-usgs**
remove unneeded file

Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

**Review by kkehl-usgs — APPROVED** [review]

**Review by orosz-usgs — APPROVED** [review]
Looks good, nothing stood out for me. I have quite a bit less experience in AWS, so it would
be good if Sam also looks at it.

File: package.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "name": "aquarius-ts-capture-trigger",
   "description": "Deployment via serverless",
   "dependencies": {
-    "serverless": "1.77.1",
+    "serverless": "2.1.1",
+    "serverless-pseudo-parameters": "2.5.0",
```
Comment (ssoper-usgs) [inline package.json:6]: Just clarifying here - does `serverless-psuedo-parameters` need to be specified here or does it come along with serverless now? I remember a week or so ago we thought maybe it was just bundled with serverless.

File: serverless.yml [hunk 2]
```diff
@@ -151,4 +152,4 @@ package:
     - .dockerignore
     - Jenkinsfile
     - package.json
-    - package-lock.json
+    - package-lock.json
```
Comment (ssoper-usgs) [inline serverless.yml]: need a newline here

**Review by ssoper-usgs — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (ayan-usgs) [reply package.json:6]: It does. Perhaps our investigation last week discovered spurious data.

**Pull request merged**